db/seeds.rb

```diff
@@ -50,13 +50,17 @@
   User.create!(first_name: 'Remote Clinic', last_name: 'Lawyer', email: 'remote_clinic_lawyer@example.com', community_id: nyc_community.id, phone: '888 888 8888', password: 'Password1234', password_confirmation: 'Password1234', invitation_accepted_at: Time.now, volunteer_type: 2, role: 0, pledge_signed: true, remote_clinic_lawyer: true)
 
   #Some additional NYC volunteer users
-  20.times do
+  20.times do |index|
+    Timecop.travel(index.days.ago)
```

Comment:
Nice!  I dig it!

Reply:
i did the same in Artful.ly recently to stagger ticket sales, it's a pretty nifty trick.